src/tls/cockpit-certificate-helper.in

```diff
@@ -10,15 +10,15 @@ COCKPIT_WS_CERTS_D="${COCKPIT_CONFIG}/ws-certs.d"
 COCKPIT_RUNTIME_DIR="/run/cockpit"
 
 install_cert() {
-    local destination="${COCKPIT_WS_CERTS_D}/$1"
+    destination="${COCKPIT_WS_CERTS_D}/$1"
```

Comment:
FTR, an alternative fix is to use `#!/bin/bash` explicitly.

Reply:
*grumble* is there some way that we can explain to coverity that `local` is implemented by every single reasonable shell on this planet?  I use it a lot, and I'd rather not stop.

Reply:
I suppose someone who has /bin/sh pointing to tcsh or fish will be in a world of pain indeed :grin: And obviously `dash` understands it just fine. But as I said, explicitly running this through bash seems fine to me -- this is about as far away from "performance critical" as you can get.

Reply:
Ok, do we agree to use `#!/bin/bash`? Might it happen that one would not have bash and this script would fail?

Reply:
Not on any OS that we support, anyway. We use bash in quite a number of places already.

Reply:
No.  I'm sorry, I don't agree.  We shouldn't be making our code worse to make Coverity happy.  Please find another way to make this problem go away.

Reply:
I don't think we are making it worse. Here are options we have:
1. Keep it as is and just waive it in coverity. However we use something that is not officially documented and we just assume that any reasonable system will work. That does not seem correct.
2. We don't use `local`. That possibly would make the code "worse just to make coverity happy".
3. We use `#!/bin/bash` and keep `local`. That way we use something that is officially supported. We won't break anything as we use `#!/bin/bash` in other places in Cockpit already.

So I don't see how doing it correctly is "worse to make Coverity happy".